docs/hub-deployment-guide/configure-auth/github-orgs.md

```diff
@@ -97,7 +97,6 @@ You can remove yourself from the org once you have confirmed that login is worki
           GitHubOAuthenticator:
             oauth_callback_url: https://{{ HUB_DOMAIN }}/hub/oauth_callback
             allowed_organizations:
-              - 2i2c-org:hub-access-for-2i2c-staff
```

Comment:
Since we're updating the GitHub authentication example to not include the 2i2c org, let's also update the example in https://infrastructure.2i2c.org/hub-deployment-guide/configure-auth/cilogon/#configure-the-authenticator to not include 2i2c in the `allowed_domains`.

Reply:
Resolved via 53c8e49